src/ssd1306-element-element.ts

```diff
@@ -1,12 +1,44 @@
 // Reference: https://cdn-learn.adafruit.com/assets/assets/000/036/494/original/lcds___displays_fabprint.png?1476374574
 import { customElement, html, LitElement, property, SVGTemplateResult } from 'lit-element';
 
-type NumTuple = [number, number];
+type Dimensions = [number, number];
 
 @customElement('wokwi-ssd1306-element')
 export class Ssd1306ElementElement extends LitElement {
-  @property() dimensions: NumTuple = [141, 108];
-  @property() bitmap: any; // size should be 128 x 64
+  @property() dimensions: Dimensions = [141, 108];
+  @property() imageData: ImageData = new ImageData(128, 64);
+
+  putGivenImageData() {
+    const canvas: HTMLCanvasElement | null = document.querySelector('canvas');
```

Comment:
With `lit-element`, the components are enclosed in Shadow DOM, so there is an API for accessing them, called `shadowRoot`. You can see an example here: https://lit-element.polymer-project.org/guide/lifecycle#examples

Reply:
thanks :) it works